package.json

```diff
@@ -20,6 +20,10 @@
   ],
   "scripts": {
     "admin": "node ./bin/devkit-admin",
```

Comment:
Nit: would be nice to have `"bazel": "bazel",`

Reply:
`npx bazel`?

Reply:
Having the script proper is better I feel. It makes it pretty obvious that it's the right way of using `bazel`, whether via `yarn` or `npm`, and it's more ergonomic for scripts too. I don't know if all the docker images we use have `npx` either, but I guess they would.

Reply:
It comes with npm.  If you’re only using yarn you should be able to just do ‘yarn bazel’

Reply:
yeah you can `yarn bazel` if you want to pick your targets.